templates/base.html

```diff
@@ -233,10 +233,15 @@
         </ul>
 
         <span id="user-links">
-            <a id="flag_vi" href="#" onclick="submmit_language('vi')" style="margin-right: 3px;">
+            <a href="/tickets/new" style="margin-right: 5px;">
```

Comment:
can we make it to /tickets/news?issue_url=`current url`. 

That will make user don't have to type the url of the current page.

Reply:
Good idea =))
Can u do that right now?